appeals/web/src/server/lib/middleware/add-apiclient-to-request.js

```diff
@@ -14,7 +17,12 @@ const instance = got.extend({
 
 const getInstance = (/** @type {string} */ userId) =>
 	instance.extend({
+		retry: {
```

Comment:
This should be good for Appeals. It's a got instance injected in the request through middleware, so it can be configured per user.